src/EditorFeatures/TestUtilities/Diagnostics/DiagnosticProviderTestUtilities.cs

```diff
@@ -36,14 +36,14 @@ public static async Task<IEnumerable<Diagnostic>> GetAllDiagnosticsAsync(Diagnos
 
         public static async Task<IEnumerable<Diagnostic>> GetDocumentDiagnosticsAsync(DiagnosticAnalyzer workspaceAnalyzerOpt, Document document, TextSpan span, Action<Exception, DiagnosticAnalyzer, Diagnostic> onAnalyzerException = null, bool logAnalyzerExceptionAsDiagnostics = false, bool includeSuppressedDiagnostics = false)
         {
-            var testDriver = new TestDiagnosticAnalyzerDriver(document.Project, workspaceAnalyzerOpt, onAnalyzerException, logAnalyzerExceptionAsDiagnostics, includeSuppressedDiagnostics);
```

Comment:
Is `logAnalyzerExceptionAsDiagnostics` no longer used? Should we remove the unused parameter?

Reply:
will do.